tests/test_torch_agent.py

```diff
@@ -75,6 +75,9 @@ def dictionary_class():
         """Replace normal dictionary class with mock one."""
         return MockDict
 
+    def build_model(self):
+        self.model = None
```

Comment:
This one seems off :D

Reply:
this was just because the method was marked abstract before but can remove this now

Reply:
removed

Reply:
I guess it needs to be return None instead?